lib/ansible/modules/network/nxos/nxos_nxapi.py

```diff
@@ -162,37 +162,44 @@ def check_args(module, warnings):
 
 
 def map_obj_to_commands(want, have, module):
-    commands = list()
+    send_commands = list()
+    commands = dict()
```

Comment:
I'm not understanding what the point of changing commands to a dict here is, can you explain why the change?